tests/examples/wallet/test_wallet.py

```diff
@@ -1,73 +1,106 @@
 import pytest
 
+from eth_keys import KeyAPI
```

Comment:
You can also use `from eth_keys import keys` and remove anywhere that you are doing `keys = KeysAPI()`